integration-tests/tests.ts

```diff
@@ -1,30 +1,60 @@
-import {
-  test,
-  expect,
-  ConsoleMessage,
-  Page,
-  TestInfo,
-  BrowserContext,
-} from "@playwright/test";
+import { test, expect, ConsoleMessage, Page, TestInfo } from "@playwright/test";
 import fs from "fs";
 
-const BASE_URL = process.env.BASE_URL;
-const BWD_BASE_URL = process.env.BWD_BASE_URL;
-const options = {
-  baseURL: BASE_URL,
-  bwdBaseURL: BWD_BASE_URL,
-};
```

Comment:
I previously added `bwdBaseUrl` to this `options` thinking it was something general that _we_ defined, rather than something Playwright specific. Rather, baseURL is something specific to Playwright that we're able to override. This gets rid of that, rather inlining the object into the `.use()` below.